core/consensus/component.go

```diff
@@ -342,13 +342,18 @@ func (c *Component) handle(ctx context.Context, _ peer.ID, req proto.Message) (p
 
 	pbMsg, ok := req.(*pbv1.ConsensusMsg)
 	if !ok {
-		return nil, false, errors.New("invalid consensus message type")
+		return nil, false, errors.New("invalid consensus message")
 	}
 
 	if pbMsg.Msg == nil || pbMsg.Msg.Duty == nil {
 		return nil, false, errors.New("invalid consensus message fields")
 	}
 
+	typ := qbft.MsgType(pbMsg.Msg.GetType())
+	if !typ.Valid() {
+		return nil, false, errors.New("invalid consensus message type", z.Str("type", typ.String()))
```

Comment:
`typ.String()` doesn't make sense for invalid types, suggest just using `int` value